database/models/models.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+type PlayerSave struct {
+	DiscordId    int64             `bson:"discord_id"`
+	LastUsername string            `bson:"last_username"`
+	Money        int64             `bson:"money"`
+	Resources    map[string]uint32 `bson:"resources"`
+	Progress     Progress          `bson:"progress"`
+}
+
+type Progress struct {
+	Planet Planet `bson:"planet"`
+	Quest  uint16 `bson:"quest"`
```

Comment:
Se a quest for salva com o progresso, sugiro ser: Quest [][2]uint16.
O primeiro valor representa a Quest e o segundo o progresso dentro da quest.